.cascade-code/_cascade/formats/modular/widgets/Funnels_2up

```diff
@@ -72,6 +71,18 @@
   </div>
 #end
 
+#macro( buildFunnelBlockLink $isLink $headlineLink )
+    #if ( $isLink )
+      <a href="${_EscapeTool.xml($headlineLink)}">
+        #buildFunnelBlockFeatureImage( $imageUrl $altText )
+        #buildFunnelBlockHeader( $headlineText )
+      </a>
```

Comment:
Has simply wrapping blocks in an anchor tag been tested? I just had to undo something like this in Blogs because it was causing markup and WAVE errors.